R/lazy.R

```diff
@@ -118,7 +117,7 @@ print.tbl_connect <- function(x, ..., n = NULL) {
 
 #' @export
 as.data.frame.tbl_connect <- function(x, row.names = NULL, optional = NULL, ..., n = Inf) {
-  as.data.frame(collect(x, n = n))
+  as.data.frame(dplyr::collect(x, n = n))
 }
```

Comment:
Might be worth an explicit comment that we don't need to condition on this because the only way to have `tbl_connect` available at all is if dplyr is already available.